frontend/src/routes/(app)/project/[owner]/[project]/+layout.svelte

```diff
@@ -14,23 +16,27 @@
 
 	export let data;
 
-	$: setupProject(data);
+	project.set(data.project);
+	rowsPerPage.set(data.project.samplesPerPage ?? 10);
+	slices.set(data.slices);
+	columns.set(data.columns);
+	models.set(data.models);
+	if (data.models.length > 0) {
+		model.set(data.models[0]);
+	}
+	metrics.set(data.metrics);
+	if (data.metrics.length > 0) {
+		metric.set(data.metrics[0]);
+	}
+	folders.set(data.folders);
+	tags.set(data.tags);
```

Comment:
Not sure about this: previously was reactive to changes in data. But maybe reloads anyway if data changes?

Reply:
Yeah if data changes it was reloaded.